package.json

```diff
@@ -84,33 +81,22 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "gh-pages": "^3.1.0",
-    "html-webpack-plugin": "^5.3.2",
     "husky": "~3.1",
     "identity-obj-proxy": "^3.0.0",
-    "image-webpack-loader": "^7.0.1",
     "jest": "^26.6.3",
     "lint-staged": "~9.4",
-    "mini-css-extract-plugin": "^1.3.8",
     "npm-run-all": "~4.1",
-    "nyc": "^15.1.0",
-    "postcss": "^8.3.9",
-    "postcss-loader": "^6.1.1",
     "prettier": "^2.0.5",
     "react": "^16.14.0",
     "react-dom": "^16.13.0",
     "rimraf": "~3.0",
-    "snowpack": "^3.2.2",
     "ts-jest": "^26.1.1",
     "ts-node": "^9.1.1",
     "typescript": "^4.5.2",
-    "webpack": "^5.52.0",
```

Comment:
Have you verified that running the storybook command still works? I ran into an issue at work a couple of months ago where removing webpack resulted in Storybook breaking because it needed webpack as a dependency.

Reply:
Yes, `npm run storybook` works.

Yes, me too when I tried to remove webpack. I had to run a fresh `npm i` (with webpack removed from package.json) so that  the webpack dependency works out.